releases/release-1.24/release-notes/release-notes-draft.json

```diff
@@ -3078,30 +3786,1041 @@
+    "author": "ahg-g",
+    "author_url": "https://github.com/ahg-g",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/108648",
+    "pr_number": 108648,
+    "areas": [
+      "test"
+    ],
+    "kinds": [
+      "feature"
+    ],
+    "sigs": [
+      "scheduling",
+      "storage",
+      "testing"
+    ],
+    "feature": true,
+    "duplicate": true
+  },
+  "108662": {
+    "commit": "9642a1f84b97582c29142a9c04d0c25c5632ad08",
+    "text": "Fix to allow fsGroup to be applied for CSI Inline Volumes",
+    "markdown": "Fix to allow fsGroup to be applied for CSI Inline Volumes ([#108662](https://github.com/kubernetes/kubernetes/pull/108662), [@dobsonj](https://github.com/dobsonj)) [SIG Storage]",
+    "author": "dobsonj",
+    "author_url": "https://github.com/dobsonj",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/108662",
+    "pr_number": 108662,
+    "kinds": [
+      "bug"
+    ],
+    "sigs": [
+      "storage"
+    ]
+  },
+  "108747": {
+    "commit": "c28cfea07d1db48764656a2b241e7266e17cdbbc",
+    "text": "The `TopologyAwareHints` feature gate is now enabled by default. This will allow users to opt-in to Topology Aware Hints by setting the `service.kubernetes.io/topology-aware-hints` on a Service. This will not affect any Services without that annotation set.",
+    "markdown": "The `TopologyAwareHints` feature gate is now enabled by default. This will allow users to opt-in to Topology Aware Hints by setting the `service.kubernetes.io/topology-aware-hints` on a Service. This will not affect any Services without that annotation set. ([#108747](https://github.com/kubernetes/kubernetes/pull/108747), [@robscott](https://github.com/robscott)) [SIG Network]",
+    "author": "robscott",
+    "author_url": "https://github.com/robscott",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/108747",
+    "pr_number": 108747,
+    "kinds": [
+      "bug",
+      "cleanup"
+    ],
+    "sigs": [
+      "network"
+    ],
+    "duplicate_kind": true
+  },
+  "108761": {
+    "commit": "aface3ab99b5c8c6ac8c4892c8019742075a461d",
+    "text": "Set PodMaxUnschedulableQDuration as 5 min.",
+    "markdown": "Set PodMaxUnschedulableQDuration as 5 min. ([#108761](https://github.com/kubernetes/kubernetes/pull/108761), [@denkensk](https://github.com/denkensk)) [SIG Scheduling]",
+    "author": "denkensk",
+    "author_url": "https://github.com/denkensk",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/108761",
+    "pr_number": 108761,
+    "kinds": [
+      "feature"
+    ],
+    "sigs": [
+      "scheduling"
+    ],
+    "feature": true
+  },
+  "108769": {
+    "commit": "4d08582d1fa21e1f5887e73380001ac827371553",
+    "text": "kubeadm: fix a bug where Windows nodes fail to join an IPv6 cluster due to preflight errors",
```

Comment:
fix -> fixed

Reply:
i don't think k8s has conventions for the language in release notes, but for Git commits that would be imperative mood:
https://medium.com/@rahul.allada.rao/imperative-mood-for-git-commit-message-72b61418271e
https://cbea.ms/git-commit/#imperative